src/autolabel/models/palm.py

```diff
@@ -120,17 +135,21 @@ def _label(self, prompts: List[str]) -> RefuelLLMResult:
             prompts = [[HumanMessage(content=prompt)] for prompt in prompts]
 
         try:
+            start_time = time()
             result = self._label_with_retry(prompts)
+            end_time = time()
             for generations in result.generations:
                 for generation in generations:
                     generation.text = generation.text.replace(
                         self.SEP_REPLACEMENT_TOKEN, "\n"
                     )
             return RefuelLLMResult(
-                generations=result.generations, errors=[None] * len(result.generations)
+                generations=result.generations,
+                errors=[None] * len(result.generations),
+                latencies=[end_time - start_time] * len(result.generations),
             )
         except Exception as e:
-            self._label_individually(prompts)
```

Comment:
this might no longer be needed given we are doing the lableing with chunk size as 1 - independent of this PR